javalib-ext-dummies/src/main/scala/java/util/Locale.scala

```diff
@@ -40,9 +40,9 @@ final class Locale(
     that match {
       case that: Locale =>
         getLanguage() == that.getLanguage() &&
-          getCountry() == that.getCountry() &&
-          getVariant() == that.getVariant() &&
-          extensions == that.extensions
+        getCountry() == that.getCountry() &&
+        getVariant() == that.getVariant() &&
```

Comment:
Do you maybe know if there is a setting that would allow us to stick with the previous layout? It actually made a lot of sense to indent the continued condition to make it more visible and to not mismatch it with some other expression

Reply:
I talked to the powers at be but apparently this is top-level in a case and the format is different - https://github.com/scalameta/scalafmt/pull/3018 I don't really understand why but ...

Reply:
it's top level but you can exclude `&&`

Reply:
I see that you.

Reply:
@kitbellew found the rule - https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md#long-expressions-with-binary-operators